templates/iot/examples/iot-config.yaml

```diff
@@ -10,7 +10,7 @@ spec:
           external:
             username: registry
             password: user12
-            url: jdbc:postgresql://postgresql.device-registry-storage.svc/device-registry
```

Comment:
JB fixed this in order to align this with the documentation. I think we should leave it that way.

Reply:
I would rather change the docs as how it is now, the getting started guide doesn't get you started :) (And you need figure on your own what's wrong)